components/observations/ObservationsListView.tsx

```diff
@@ -171,7 +206,10 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
 
   // the displayed observations need to match all filters - for instance, if a user chooses a zone *and*
   // an observer type, we only show observations that match both of those at the same time
-  const resolvedFilters = useMemo(() => (mapLayer ? filtersForConfig(mapLayer, filterConfig, additionalFilters) : []), [mapLayer, filterConfig, additionalFilters]);
+  const resolvedFilters = useMemo(
```

Comment:
I think you need to run `yarn prettify` and/or configure your editor to use our prettier config for formatting diffs like this.